### ISSUE TITLE
Test of direct, parallel FFT fails

### ISSUE THREAD

**Issue by sbrisard** [sbrisard/janus#12]
Commit 32c3b76 reveals the following bug

```
______________________________ test_r2c[shape809] ______________________________

shape = (33, 33, 33)

    @pytest.mark.parametrize('shape', SHAPES)
    def test_r2c(shape):
        comm = MPI.COMM_WORLD
        root = 0

        janus.fft.parallel.init()
>       pfft = janus.fft.parallel.create_real(shape, comm)

test_parallel_fft.py:22: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

>   ???

_parallel_fft.pyx:12: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

>   ???

_parallel_fft.pyx:43: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

>   ???
E   AttributeError: 'janus.fft.serial._serial_fft._RealFFT3D' object has no attribute 'oshape0'

_serial_fft.pyx:178: AttributeError

```

**Comment by sbrisard** [comment]
Corrected typo in 08f7554aa1a3.

**Comment by sbrisard** [comment]
Made test of forward, parallel Fourier transform less stringent in 80b84029f7b1.
